Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -244,7 +258,7 @@ def search_command(proxies):
     index = demisto.args().get('index')
     query = demisto.args().get('query')
     fields = demisto.args().get('fields')  # fields to display
-    explain = 'true' == demisto.args().get('explain')
+    explain = demisto.args().get('explain') == 'true'
```

Comment:
IT's args from the user. You don't know if they'll enter `True`.
```suggestion
    explain = demisto.args().get('explain').lower() == 'true'
```